server/src/routes/clientRouter.js

```diff
@@ -6,6 +6,7 @@ const router = new express.Router();
 
 const clientRoutes = [
   "/",
+  "/user/page",
```

Comment:
`page` is too generate. `/user-profile` or `/profile` are more ideal. Only add an additional `/` if there are other resources of information we need in the URL.